.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: 'Feature Request/Idea:'
+labels: ''
+assignees: ''
+
+---
+<!--
+Thank you for contributing to the Dataverse Project!
+
+Your idea or feature request might already be captured in the Dataverse issue tracker on GitHub but if not, the best way to bring it to the community's attention is by posting on the [dataverse-community Google Group](https://groups.google.com/group/dataverse-community) or bringing it up on a [Community Call](https://dataverse.org/community-calls). You're also welcome to make some noise in chat.dataverse.org or cram your idea into 280 characters and mention @dataverseorg on Twitter. To discuss your idea privately, please email it to support@dataverse.org
```

Comment:
I think 12-15 are good (and I know it's not just the issue tracker in play like in the bug report) but I think we should remove in the first iteration. We can point people to the contributing guide (as you've done) for more details.

Reply:
Makes total sense.